e2e/prisoners.e2e-spec.ts

```diff
@@ -0,0 +1,18 @@
+import {browser, by, element, ExpectedConditions as EC} from 'protractor';
+
+fdescribe('Prisoner manifest', () => {
+  it('should redirect to the login', () => {
+    browser.waitForAngularEnabled(false);
+    browser.get('/prisoners');
+    expect(browser.getCurrentUrl()).toBe(browser.baseUrl + '/assets/login.html');
+  });
+
+  it('should log in with fake credentials', () => {
+    element(by.css('.user')).sendKeys('foo');
+    element(by.css('.password')).sendKeys('bar');
+    element(by.css('button#login')).click();
+    browser.waitForAngularEnabled(true);
+    browser.sleep(300);
```

Comment:
We shouldn't encourage browser.sleep - could we do something with browser.wait instead?

Reply:
done.